lms/djangoapps/commerce/api/v0/views.py

```diff
@@ -149,7 +150,13 @@ def post(self, request, *args, **kwargs):  # lint-amnesty, pylint: disable=unuse
                     announcement=course_announcement
                 )
             log.info(msg)
-            self._enroll(course_key, user, default_enrollment_mode.slug)
+            try:
+                self._enroll(course_key, user, default_enrollment_mode.slug)
+            except (EnrollmentNotAllowed, InvalidEnrollmentAttribute) as e:
+                log.exception("Enrollment failed: %s", str(e))
+                error_msg = f"Failed to enroll user {user.username} in course {course_id}: {str(e)}"
+                return DetailResponse(error_msg, status=403)
```

Comment:
```suggestion
                return DetailResponse(error_msg, status=HTTP_403_FORBIDDEN)
```
To avoid magic numbers, I suggest you import from rest_framework.status the status code you need.
More details: https://github.com/encode/django-rest-framework/blob/master/rest_framework/status.py#L30

Reply:
@efortish, you marked resolve, but I didn't see your changes. I still see the 403 code.

Reply:
Sorry, it was fixed!